lib/action_policy/lookup_chain.rb

```diff
@@ -112,19 +109,19 @@ def policy_class_name_for(record)
     }
 
     # Infer from passed symbol
-    SYMBOL_LOOKUP = ->(record, namespace: nil, **) {
+    SYMBOL_LOOKUP = ->(record, namespace: nil, strict_namespace: false, **) {
       next unless record.is_a?(Symbol)
 
       policy_name = "#{record.camelize}Policy"
-      lookup_within_namespace(policy_name, namespace) || policy_name.safe_constantize
+      lookup_within_namespace(policy_name, namespace, strict_namespace)
```

Comment:
Let's make it a  `strict:` keyword argument.

Reply:
@palkan thanks for the feedback. just committed these changes :+1: